src/main/java/games/strategy/triplea/util/TransportUtils.java

```diff
@@ -314,7 +317,7 @@ private static boolean containsEquivalentUnit(final Unit unit, final Collection<
   }
 
   private static Unit getEquivalentUnit(final Unit unit, final Collection<Unit> units) {
-    return units.stream().filter(u -> u.isEquivalent(unit)).findFirst().get();
+    return units.stream().filter(unit::isEquivalent).findAny().get();
```

Comment:
This code shouldn't change the behaviour at all.
Previously, if unit was null, all elements would've been filtered out resulting in a NoSuchElementException, now a NullPointerException would be thrown

Reply:
Right. And I doubt we are specifically catching NoSuchElementException anywhere.